components/tabs/src/storage.rs

```diff
@@ -578,50 +615,76 @@ impl TabsStorage {
         let tx = conn.unchecked_transaction()?;
 
         // Insert new remote tabs into a temporary table
-        conn.execute(
-            "CREATE TEMP TABLE if not exists new_remote_tabs (device_id TEXT, url TEXT)",
-            [],
+        conn.execute_batch(
+            "
+            CREATE TEMP TABLE if not exists new_remote_tabs (device_id TEXT, url TEXT);
+            DELETE FROM new_remote_tabs;
+            CREATE TEMP TABLE if not exists new_remote_tabs_without_inactive (device_id TEXT);
```

Comment:
```suggestion
            CREATE TEMP TABLE IF NOT EXISTS new_remote_tabs (device_id TEXT, url TEXT);
            DELETE FROM new_remote_tabs;
            CREATE TEMP TABLE IF NOT EXISTS new_remote_tabs_without_inactive (device_id TEXT);
```

Nit: Let's capitalize keywords (I know the `if not exists` was lowercased before).

Follow-up / food for thought: You could turn this into a single table with an `ifempty(url, '')` and a `command` type—basically, mirroring how `remote_tab_commands` is set up! That might make it a little easier to do the `DELETE FROM remote_tab_commands` below?—but that's more of my own style preference; functionally, the way you have this now is also good!